ovs/vswitch.go

```diff
@@ -88,6 +88,9 @@ func (v *VSwitchService) ListBridges() ([]string, error) {
 		return nil, err
 	}
 
+	if string(output) == "" {
+		return []string{}, err
```

Comment:
Let's return `nil, nil` in this case.

Reply:
Sure, let me fix this.

Reply:
It's good to use `nil` to instead `[]string{}` ?
I think ListBridges() can return empty array when someone try to call it to get bridges length.

Reply:
Yep, so nil is fine because whenever someone is working with a slice, they are either iterating it or trying to access a specific element, which they should only do after making a bounds check.

For the purposes, `nil` and `[]string{}` are effectively equivalent, but I prefer nil because it truly conveys "nothing" is returned.

Does that make sense?

Reply:
Get it. Let me fix for this.